package.json

```diff
@@ -21,22 +22,29 @@
   },
   "homepage": "https://github.com/Visual-Testing-for-Android-Apps/web#readme",
   "devDependencies": {
+    "express-pino-logger": "^6.0.0",
+    "nodemon": "^2.0.7",
+    "pino-colada": "^2.1.0",
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
     "@babel/cli": "^7.13.14",
     "@babel/core": "^7.13.14",
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-react": "^7.13.13",
+    "axios": "^0.21.1",
     "babel-loader": "^8.2.2",
+    "cors": "^2.8.5",
     "css-loader": "^5.2.1",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.3.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-dropzone": "^11.3.2",
     "react-hot-loader": "^4.13.0",
+    "react-recaptcha": "^2.3.10",
```

Comment:
You're not using this right? If not, uninstall it `npm uninstall react-recaptcha`